bounce.py

```
@@ -26,20 +26,26 @@
+            extracted = tldextract.extract(domain)
+            domain = f"{extracted.domain}.{extracted.suffix}"
+            logging.debug(f"Valid domain extracted: {domain}")
+            domain_counter_stat.labels(domain=domain, status='valid').inc()
+            redirect_url = bounce_redirect_url + domain
+        else:
+            logging.warning(f"Invalid domain: {domain}, redirecting to root.")
+            domain_counter_stat.labels(domain="root", status='invalid').inc()
+
+        logging.debug(f"Redirecting to URL: {redirect_url}")
+        return redirect(redirect_url, code=301)
+    except Exception as e:
```

Ruff (0.7.0)

Do not catch blind exception: `Exception`

(BLE001)